Sources/CocoaLumberjack/Extensions/DDDispatchQueueLogFormatter.m

```diff
@@ -16,11 +16,23 @@
 #import <CocoaLumberjack/DDDispatchQueueLogFormatter.h>
 #import <pthread/pthread.h>
 #import <stdatomic.h>
+#import <sys/qos.h>
 
 #if !__has_feature(objc_arc)
 #error This file must be compiled with ARC. Use -fobjc-arc flag (or convert project to ARC).
 #endif
 
+static const char* _qos_string(NSUInteger qos) {
+    switch ((qos_class_t) qos) {
+        case QOS_CLASS_USER_INTERACTIVE: return "UI";
```

Comment:
What is a purpose to corrupt name?
Could consumer change default values?
Why method name is _qos_string and return values are "qos_codes"?

Reply:
Corrupted names? You mean returning "UI" instead of "USER_INTERACTIVE" for example?
_qos_string() is returning C string hence it's name.

Reply:
@jcbertin 
We have to do something with these codes.
`UN/UT/UI/IN` - not ok.

Reply:
I used those code because they are small and use extensively in WWDC keynotes. Do you want me to change it to longer version anyway?

Reply:
I agree with @lolgear that those codes are a little "cryptic". AFAIK, Apple also shows the full name in the Xcode sidebar.

Reply:
Interesting. Apple uses them as well? If so, I'd have not problem using them.

Reply:
See https://developer.apple.com/videos/play/wwdc2015/718/

Reply:
Time 10:11.

Interesting. Maybe we could, well... add "emoji" to distinct qos values?

Reply:
😋

Reply:
@ffried @jcbertin 
O!
We could save codes, but for that we should somehow tell consumer what they mean.

```objective-c
extern NSString *CCQOSStringUserInteractive= "UI";
```

What do you think?

Reply:
@lolgear yes it could clarify the meaning of those codes.

Reply:
I don't think that this should be our responsibility. If Apple uses those, they should provide them as well. So this would be something for feedbackassistant.apple.com 😉

Reply:
See c000783